### PR TITLE
Banner image component

### PR DESCRIPTION
**Pull request by OptimumCOW**
https://trello.com/c/LBO5EYsT

### CODE REVIEW

File: WhaleSpotting/react-app/src/tests/BannerImage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { BannerImage } from "../components/BannerImage";
+import { BrowserRouter as Router } from "react-router-dom";
+
+test("Renders banner image", () => {
+    render(<Router><BannerImage/></Router>);
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/tests/BannerImage.test.tsx]: can you remove the Router pls, and then tidy the imports

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/react-app/src/tests/BannerImage.test.tsx (hunk 1 above)
Comment (OptimumCOW) [reply WhaleSpotting/react-app/src/tests/BannerImage.test.tsx]: addressed.

**Comment by OptimumCOW** [comment]
added the lint rules and removed console.log statements except in Auth files where instead we have ignored those lines in the linter.

File: WhaleSpotting/react-app/.eslintrc.json [hunk 2]
```diff
@@ -36,12 +36,15 @@
         "jsx-quotes": [
             "error",
             "prefer-double"
-        ]
+        ],
+        "react/jsx-uses-react": "off",
+        "react/react-in-jsx-scope": "off",
+        "no-console": "error"
+
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/.eslintrc.json]: extra new line

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**